.github/actions/optimized_build/action.yaml

```diff
@@ -29,19 +29,14 @@ runs:
     - name: "Check for WASM binaries larger than allowed limit"
       shell: "sh"
       run: |
-        # Temporarily disable check
-        exit 0
+        large_files="`find "artifacts/" -size +3M | sed 's/ /\n/g'`"
 
-        large_files=`find "artifacts/" -size +3M`
-
-        large_files_count=`wc -l < EOF
-          ${large_files}
-          EOF`
+        large_files_count="`echo -n "${large_files}" | wc -l`"
 
         if ! test "${large_files_count}" = "0";
           then (
             (
-              echo "There are ${large_files_count} files with size larger than allowed limit!"
+              echo "There are files with size larger than allowed limit!"
```

Comment:
why don't we print the number?

Reply:
Why would we? It doesn't bring any meaningful information. On the other hand, the two lines below it I am printing the meaningful one, the file names.